examples/pubsub/README.md

```diff
@@ -41,6 +41,8 @@ const node = await Libp2p.create({
 Once that is done, we only need to create a few libp2p nodes, connect them and everything is ready to start using pubsub.
 
 ```JavaScript
+const { TextDecoder, TextEncoder } = require("util");
```

Comment:
Since node 11 it is available in the global scope like in browser, so no need to require it.

However, we use uint8arrays everywhere across the stack and it will likely be important for the users to use them in other scenarios. With this in mind, I think it is friendlier to just rely on them than to do `TextDecoder(TextDecoder.encoding).decode`. Thoughts?

Reply:
Ah actually I just noticed that you guys are also the maintainer of this library: https://www.npmjs.com/package/uint8arrays

Then I guess it'd make more sense to require the library in the example